src/System.Runtime.Analyzers/Core/SystemRuntimeAnalyzersResources.resx

```diff
@@ -307,7 +307,7 @@
     <value>A method or constructor calls a member that has an overload that accepts a System.Globalization.CultureInfo parameter, and the method or constructor does not call the overload that takes the CultureInfo parameter. When a CultureInfo or System.IFormatProvider object is not supplied, the default value that is supplied by the overloaded member might not have the effect that you want in all locales.</value>
   </data>
   <data name="SpecifyCultureInfoMessage" xml:space="preserve">
-    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'. If the result of '{2}' will be displayed to the user, specify 'CultureInfo.CurrentCulture' as the 'CultureInfo' parameter. Otherwise, if the result will be stored and accessed by software, such as when it is persisted to disk or to a database, specify 'CultureInfo.InvariantCulture'.</value>
+    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
   </data>
```

Comment:
Why was the last statement removed? This was very deliberate in the original rule, because users were confused as to what to pass.